### DISCUSSION TITLE
is it possible to support more than 10 monitors ?

### DISCUSSION THREAD

**Discussion by psychrabi** [MolotovCherry/virtual-display-rs#37] [Q&A] [answered]
I am looking to create remote workplaces using a program called [ASTER](https://ibiksoft.com) and it support up to 12 workplaces and up to 16 with beta version. I tried the current driver and i was able to use 9 virtual monitors to create 9 remote workplaces.

So, can this driver be made to support more than 10 monitors?

**Comment by MolotovCherry** [comment]
Hi! From what I'm reading, I don't think that's possible.

From what I can gather, the OS has a hard limit on how many total displays are possible. On Windows, this appears to be 10 total.

But I need to do more research into it; it's hard to find actual conclusive answers to this question. So this is not a hard answer until I investigate it. So I can't completely say yes or no until I do some debugging/research

**Reply by psychrabi** [reply]
I have tried another virtual monitor driver which could create upto 16 virtual monitors
https://github.com/ge9/IddSampleDriver

So I think the hard limit is higher than 10 monitors and may be 16  or even more.

**Reply by MolotovCherry** [reply]
I will test it out later

(To answer your question, the driver doesn't actually have a limit for anything. No limit on resolution, refresh rates, or number of monitors. It's just a hard limit coded into the control app, cause I figured 10 is more than enough)

**Reply by MolotovCherry** [reply, marked as answer]
@psychrabi Tested it out. I can increase this. Thing is, Windows sets a hard limit still, so infinite monitors is not possible, also, a computer's hardware can only handle so many displays. Even on a 3080Ti I can feel some slowdown due to this (this is NOT the drivers fault)

I also want to note that the limit of monitors is actually a TOTAL amount in the entire system. I'll account for this fact in my gui next time by only allowing MAX-current number of physical monitors

Do you have hardware that can handle that kind of load?

(Also a sidenote, but the driver you linked to has a critical bug in its processing which causes insane slowdown with > 1 monitor - a bug which is not present in my driver)

**Reply by psychrabi** [reply]
Thabk you for testing it and looking forward to next release 

And, Yes I do have system I can test the so many virtual monitors.
I had been trying different drivers to se which is the best one.

And yes the driver has major bug when using with nvidia cards.
I had no issue with amd graphics card and Intel and amd integrated graphics
